### PR TITLE
feat: Added hamburger menu + Added some dev resources and App ideas

### PR DESCRIPTION
**Pull request by yogesh7401**
## Feature
- Added hamburger menu
## Resources
- Story set
- Drawkit
- Iconfinder
## App Ideas
- Counter Timer
- Calendar app
- Bus Booking system
- Hospital management system
- Online Exam System
- Online Chat system

### CODE REVIEW

**Review by anburocky3 — APPROVED** [review]

File: components/TheHeader.vue [hunk 1]
```diff
@@ -13,27 +13,38 @@ const route = useRoute()
...
   isLangDialogOpened.value = !isLangDialogOpened.value
+  isToggleOpened.value = false
 }
 const toggleMenu = () => {
   isThemeDialogOpened.value = false
   isLangDialogOpened.value = false
+  isToggleOpened.value = false
   isMenuOpened.value = !isMenuOpened.value
 }
+const toggleHamburgerMenu = () => {
```
Comment (anburocky3) [inline components/TheHeader.vue:36]: What is this? Can you explain in detail.

**Comment by anburocky3** [comment]
Menu link can be decent. (It is too bright)

File: components/TheHeader.vue (hunk 1 above)
Comment (yogesh7401) [reply components/TheHeader.vue:36]: To toggle mobile menu by clicking hamburger menu

**Comment by anburocky3** [comment]
Thank you for your contribution. Merged.

**Pull request merged**